src/Microsoft.SqlTools.ServiceLayer/Metadata/Contracts/GenerateServerContextualizationCompleteNotification.cs

```diff
@@ -0,0 +1,36 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using Microsoft.SqlTools.Hosting.Protocol.Contracts;
+
+namespace Microsoft.SqlTools.ServiceLayer.Metadata.Contracts
+{
+    public class GenerateServerContextualizationCompleteParams
+    {
+        /// <summary>
+        /// The URI identifying the owner of the connection
+        /// </summary>
+        public string OwnerUri { get; set; }
```

Comment:
this isn't getting set when there's an error. Should this be nullable?

Reply:
I was thinking that it would default to an empty string, but I'm thinking that setting the owner URI is a better way to go.